Update backup-and-restore.md for portainer (#11393)
immich-app/immich: docs/docs/administration/backup-and-restore.md

      - database
```

:::info

> For portainer, do not forgot to replace .env with stack.env that need to use environment variables in the web editor.

Reviewer: ```suggestion
For Portainer, remember to replace `.env` with `stack.env`, which needs to use environment variables in the web editor.
```
